libs/types/src/domain_address.rs

```diff
@@ -53,6 +53,12 @@ pub enum DomainAddress {
 	EVM(EVMChainId, [u8; 20]),
 }
 
+impl Default for DomainAddress {
```

Comment:
Super NIT: add `#[default]` to the variant

Reply:
That can only be used with unit type enums :(

Reply:
Ohh, didn't know, thanks 👍🏻